### PR TITLE
Verify fas improvements

### PR DESCRIPTION
**Pull request by lbarcziova**
I tested `/packit verify-fas` in packit/notifications#429, works just fine, only the message needs to be adjusted for staging.

Also, I added the check so that only the person who triggered the installation can be verified.

---

RELEASE NOTES BEGIN
N/A
RELEASE NOTES END

### CODE REVIEW

File: packit_service/worker/handlers/forges.py [hunk 1]
```diff
@@ -181,6 +187,22 @@ def run(self) -> TaskResults:
...
+        if original_sender_login != self.sender_login:
+            msg = (
+                "Packit verification comment command not created by the person who "
+                "installed the application."
+            )
+            logger.debug(msg)
+            self.issue.comment(msg)
+            return TaskResults(success=True, details={"msg": msg})
+
+        return self.verify(namespace=namespace, fas_account=fas_account)
```
Comment (TomasTomecek) [inline packit_service/worker/handlers/forges.py:199]: agreed that the use case for doing the verification for someone else sounds iffy

**Review by TomasTomecek — APPROVED** [review]
+1 No need to update the release notes.

**Comment by lbarcziova** [comment]
recheck

**Comment by lbarcziova** [comment]
regate

**Pull request merged**